packages/stacked_firebase_auth/lib/src/firebase_authentication_service.dart

```diff
@@ -317,6 +317,23 @@ class FirebaseAuthenticationService {
     );
   }
 
+    /// Phone Number Login
+  Future<ConfirmationResult> signInWithPhoneNumber(String phoneNumber) async {
+    return firebaseAuth.signInWithPhoneNumber(phoneNumber);
+  }
+
+  Future<FirebaseAuthenticationResult> verifyOtp(
+      ConfirmationResult confirmationResult, String otp) async {
+    try {
+      UserCredential userCredential = await confirmationResult.confirm(otp);
+      return FirebaseAuthenticationResult(user: userCredential.user);
+    } catch (e) {}
+    return FirebaseAuthenticationResult.error(
```

Comment:
Place it inside the catch closer and add the error to the end of the error message